app/stores/toplevel/ProfileStore.js

```diff
@@ -302,7 +302,7 @@ export default class ProfileStore extends Store {
   };
 
   _redirectToRoot = () => {
-    this.actions.router.goToRoute.trigger({ route: ROUTES.WALLETS.ROOT });
+    this.actions.router.goToRoute.trigger({ route: ROUTES.WALLETS.ADD });
```

Comment:
Why was this changed? Doesn't match the function name anymore.

Reply:
ProfileStore reactions were triggering some automatic redirections and I thought this would solve it, but it didn't. I changed this back to the original and I added a new action for updating locale and redirect to terms of use.